src/App.vue

```diff
@@ -157,21 +154,21 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     window.location.assign(businessUrl)
   }
 
-  private authAPIURL () {
-    return sessionStorage.getItem('AUTH_API_URL')
+  private authAPIURL (): string {
+    return sessionStorage.getItem('VUE_APP_AUTH_ROOT_API')
   }
   /**
    * Fetch Authorizations by NR number
    */
-  private getNRAuthorizations (nrNumber) {
+  private getNRAuthorizations (nrNumber: string) : Promise<any> {
     const url = nrNumber + '/authorizations'
     const config = {
-      baseURL: this.authAPIURL() + 'entities/'
+      baseURL: this.authAPIURL() + '/entities/'
```

Comment:
By convention in this project, a leading slash is not supposed to be required because all URLs should have a trailing slash.

I see that in this case, this URL doesn't have a trailing slash. Oops, missed that in the previous PR. Please ask @bsnopek-freshworks to add a trailing slash to this key in public/config/configuration.json and in the DC script.

Reply:
Thanks I pinged @bsnopek-freshworks in case he didn't see this comment.

Reply:
Done ! It'd been added into the configuration.json file and added as a new PR  (#58)

Reply:
Thanks Brian. I just merged PR #58.

Reply:
Oops, after rebase you still have to remove leading slash.

Reply:
Updated

Reply:
I realized I probably needed to add the BUSINESSES_URL to the DC script as well, so I tried to follow Brian's example.